src/pages/blog/starlingx-debian-technology-preview-in-release-7.md

```diff
@@ -0,0 +1,376 @@
+  StarlingX for production deployments. It will be the last release of a
+  CentOS7–based StarlingX.
+
+* StarlingX release 7.0 inherits the 5.10 version of the Linux kernel introduced
+  in StarlingX release 6.0.
+
+* StarlingX release 7.0 is also a technology Preview Release of Debian StarlingX
+  for evaluation purposes.
+
+* StarlingX release 7.0 release runs Debian Bullseye (11.3). It is limited in
+  scope to the AIO-SX configuration. Duplex, and standard configurations are not
+  available.
+
+See [Technology Preview Reduced Scope](#technology-preview-reduced-scope) for details.
+
+## Debian StarlingX General Availability
+
+An upcoming release will make Debian StarlingX generally available for
+production deployments.
+
+This upcoming release will run Debian Bullseye 11.3 or later with full
+functional equivalence to the CentOS-based StarlingX.
+
+The StarlingX release 7.0 Debian Technology Preview allows you to evaluate and
+prepare for the upcoming Debian-based General Availability release while
+continuing to run your production deployment on CentOS-based StarlingX. It is
+strongly recommended that you perform a complete assessment of StarlingX and
+your application running on StarlingX in a lab setting to fully understand and
+plan for any changes that may be required to your application when you migrate
+to the StarlingX Debian General Availability release in a production
+environment.
+
+Major features of Debian-based StarlingX will include:
+
+* Linux 5.10 Yocto-based kernel (see <https://www.yoctoproject.org/>).
+
+* The Yocto Project Kernel:
+
+  * tracks stable kernel updates very closely; staying very current with the
+    stable kernel,
+
+  * provides a reliable implementation of the pre-empt-rt patchset (see
+    <https://rt.wiki.kernel.org/index.php/Main_Page>), and provides predictable
+    and searchable CVE handling.
+
+    StarlingX will also leverage its existing relationships with the Yocto
+    Project to enhance development, bug fixes and other activities in the Yocto
+    Project kernel to drive StarlingX quality and feature content.
+
+* Debian Bullseye (11.3)
+
+  Debian is a well-established Linux Distribution supported by a large and
+  mature open-source community.
+
+* OSTree <https://ostree.readthedocs.io/en/stable/manual/introduction/>
+
+  OSTree provides for robust and efficient versioning, packaging and upgrading
+  of Linux-based systems.
+
+* An updated Installer to seamlessly adapt to Debian and OSTree
+
+  Updated software patching and upgrades for Debian and OSTree.
+
+# Operational Impacts
+
+The operational impact of Debian-based StarlingX is small. For a list of
+differences between this release and CentOS-based StarlingX, see
+<https://docs.starlingx.io/debian/kubernetes/operational-impacts-9cf2e610b5b3.html>
+
+
+
+# Technology Preview Reduced Scope
+
+
+The StarlingX release 7.0 Debian Technology Preview release will have reduced
```

Comment:
We are planning to publish the blog posts that describe 7.0 features and updates after the release is out. In that sense, please keep the post in present tense when you describe the changes and available functionality.